api/src/main/java/org/pmiops/workbench/cdr/dao/ConceptDao.java

```diff
@@ -41,4 +44,8 @@
     List<Concept> findConceptsMapsToParents(@Param("conceptId") long conceptId);
 
     List<Concept> findByConceptName(String conceptName);
+
+    @QueryHints(value = {@QueryHint(name = "org.hibernate.cacheable", value = "true")})
+    @Query(nativeQuery=true)
```

Comment:
Is there a reason not to inline the query SQL here?

Reply:
I guess it really doesn't perform differently. Looks like the only difference is that a NamedQuery specified on the entity class isn't coupled with a specific dao implementation. 
It also seems that i could add the query hints to the @NamedQuery annotation like so:

@NamedNativeQuery(name = "Concept.findGenderRaceEthnicityFromConcept",
        query = "select c.* from cdr.concept c where c.vocabulary_id in ('Gender', 'Race', 'Ethnicity')",
        resultClass = Concept.class,
        hints = @QueryHint(name = "org.hibernate.cacheable", value = "true"))

Reply:
OK. I have a mild preference for putting it in the @Query annotation... makes it clearer reading the DAO what SQL is being executed.

If you're not going to do that, add a comment saying where to look for the implementation of this query.

Reply:
I'm fine with @query and moving it into the dao.